### PR TITLE
Fixed some Doctests

### PR DESCRIPTION
**Pull request by Rishabhmishra328**
Fixed guide doctest.

### CODE REVIEW

File: doc/source/code_ref/instr/rhessi.rst [hunk 1]
```diff
@@ -7,8 +7,13 @@ Functions
...
...), datetime.datetime(2011, 4, 3, 0, 0), datetime.datetime(2011, 4, 4, 0, 0), datetime.datetime(2011, 4, 5, 0, 0), datetime.datetime(2011, 4, 6, 0, 0), datetime.datetime(2011, 4, 7, 0, 0), datetime.datetime(2011, 4, 8, 0, 0), datetime.datetime(2011, 4, 9, 0, 0), datetime.datetime(2011, 4, 10, 0, 0), datetime.datetime(2011, 4, 11, 0, 0), datetime.datetime(2011, 4, 12, 0, 0), datetime.datetime(2011, 4, 13, 0, 0), datetime.datetime(2011, 4, 14, 0, 0), datetime.datetime(2011, 4, 15, 0, 0), datetime.datetime(2011, 4, 16, 0, 0), datetime.datetime(2011, 4, 17, 0, 0), datetime.datetime(2011, 4, 18, 0, 0), datetime.datetime(2011, 4, 19, 0, 0), datetime.datetime(2011, 4, 20, 0, 0), datetime.datetime(2011, 4, 21, 0, 0), datetime.datetime(2011, 4, 22, 0, 0), datetime.datetime(2011, 4, 23, 0, 0), datetime.datetime(2011, 4, 24, 0, 0), datetime.datetime(2011, 4, 25, 0, 0), datetime.datetime(2011, 4, 26, 0, 0), datetime.datetime(2011, 4, 27, 0, 0), datetime.datetime(2011, 4, 28, 0, 0), datetime.datetime(2011, 4, 29, 0, 0), datetime.datetime(2011, 4, 30, 0, 0), datetime.datetime(2011, 5, 1, 0, 0)], 'orb_end': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}
```
Comment (debugger22) [inline doc/source/code_ref/instr/rhessi.rst]: What is this?

File: doc/source/code_ref/instr/rhessi.rst [hunk 2]
```diff
@@ -7,8 +7,13 @@ Functions
...
 .. autofunction:: sunpy.instr.rhessi.get_obssum_filename
 .. autofunction:: sunpy.instr.rhessi.get_obssumm_file
+Downloading file: http://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/http://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/hsi_obssumm_20110404_042.fits
+    ('/tmp/tmpFotiHI.fits', <httplib.HTTPMessage instance at 0x7f4862cd7f38>)
 .. autofunction:: sunpy.instr.rhessi.parse_obssumm_file
+Downloading file: http://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/http://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/hsi_obssumm_20110404_042.fits
```
Comment (debugger22) [inline doc/source/code_ref/instr/rhessi.rst]: Remove these too.

File: doc/source/guide/data_types/lightcurve.rst [hunk 3]
```diff
@@ -58,6 +58,69 @@ The data attribute stores the time-series measurements as a pandas
...
+    2013-07-21 23:59:59.034000  0.006254  0.695472  0.002307  0.001000
+    2013-07-21 23:59:59.034000  0.006254  0.695451  0.002310  0.001000
+    2013-07-21 23:59:59.034000  0.006255  0.695451  0.002310  0.001000
+    2013-07-21 23:59:59.034000  0.006254  0.695451  0.002310  0.000998
+    2013-07-21 23:59:59.034000  0.006254  0.695430  0.002310  0.001000
+    2013-07-21 23:59:59.034000  0.006255  0.695451  0.002313  0.001000
+    2013-07-21 23:59:59.034000  0.006255  0.695430  0.002310  0.001001
+    2013-07-21 23:59:59.034000  0.006255  0.695472  0.002310  0.000998
+    2013-07-21 23:59:59.034000  0.006255  0.695430  0.002310  0.000998
+    2013-07-21 23:59:59.034000  0.006254  0.695451  0.002310  0.001002
```
Comment (debugger22) [inline doc/source/guide/data_types/lightcurve.rst]: Probably this change is not required.

**Comment by debugger22** [comment]
Also remove this binary file. `doc/source/sunpydata.sqlite`. Maybe add this in `.gitignore`?

File: doc/source/guide/acquiring_data/database.rst [hunk 4]
```diff
@@ -175,7 +175,7 @@ syntax are introduced.
     >>> import sunpy.data.sample
     >>> database.add_from_file(sunpy.data.sample.AIA_171_IMAGE)
     >>> len(database)
-    1
+    10
```
Comment (dpshelio) [inline doc/source/guide/acquiring_data/database.rst]: Why if we add just one file `sunpy.data.sample.AIA_171_IMAGE` would we get `10`? I'm not sure that's right

File: doc/source/guide/acquiring_data/database.rst [hunk 5]
```diff
@@ -189,27 +189,7 @@ syntax are introduced.
...
-        1
-    COMMENT
-        FITS (Flexible Image Transport System) format is defined in 'Astronomy  and Astrophysics', volume 376, page 359; bibcode: 2001A&A...376..359H
-    ORIGIN
-        SDO/JSOC-SDP
-    DATE
-        2011-03-19T11:08:25
-    TELESCOP
-        SDO/AIA
+
```
Comment (dpshelio) [inline doc/source/guide/acquiring_data/database.rst]: Why did you remove this?

File: doc/source/guide/acquiring_data/database.rst [hunk 6]
```diff
@@ -337,7 +317,7 @@ values as keyword arguments to :class:`tables.DatabaseEntry` as follows:
     >>> entry in database
     True
     >>> len(database)
-    30
+    10
```
Comment (dpshelio) [inline doc/source/guide/acquiring_data/database.rst]: why are all changed to `10`? either you've run the queries and the data was not updated (maybe you didn't have connection to internet?) or there's something broken in the `database` itself...

File: doc/source/guide/acquiring_data/database.rst [hunk 7]
```diff
@@ -511,12 +488,7 @@ year:
     ...     ['id', 'observation_time_start', 'observation_time_end', 'instrument', 'wavemin', 'wavemax'])
     id observation_time_start     observation_time_end instrument wavemin wavemax
     -- ----------------------     -------------------- ---------- ------- -------
-    1  2011-03-19 10:54:00.340000 N/A                  AIA_3      17.1    17.1   
-    7  2011-03-19 10:54:00.340000 N/A                  AIA_3      17.1    17.1   
-    22 2011-05-08 00:00:00        2011-05-08 00:00:01  AIA        17.1    17.1   
-    23 2011-05-08 00:00:00        2011-05-08 00:00:01  AIA        21.1    21.1   
-    24 2011-05-08 00:00:02        2011-05-08 00:00:03  AIA        9.4     9.4    
-    25 2011-05-08 00:00:03        2011-05-08 00:00:04  AIA        33.5    33.5
+    1  2011-05-08 00:00:00    2011-05-08 00:00:01  AIA        17.1    17.1    N/A  No
```
Comment (dpshelio) [inline doc/source/guide/acquiring_data/database.rst]: What are the last two fields in the results: `N/A No`?

File: doc/source/guide/acquiring_data/vso.rst [hunk 8]
```diff
@@ -57,7 +57,7 @@ response objects, each one of which is a record found by the VSO. You can find h
 records were found by typing
 
     >>> len(qr)
-    122
+    3549
```
Comment (dpshelio) [inline doc/source/guide/acquiring_data/vso.rst]: There's something wrong in that result... SoHO/EIT doesn't get that much data in one day.

File: doc/source/guide/data_types/lightcurve.rst (hunk 3 above)
Comment (dpshelio) [inline doc/source/guide/data_types/lightcurve.rst]: agree with @debugger22

**Comment by dpshelio** [comment]
I agree with @debugger22 also on the `sqlite` file. Adding it to the `.gitignore` will avoid to add it by mistake. Since many of the changes you propose in the documentation are not matching with the actual results I would vote to close this PR. @Cadair, @ayshih  what do you think?

Thanks @debugger22 for the review!

**Comment by ayshih** [comment]
It's silly to have a `.gitignore` entry for just this specific `sqlite` file.  It appears to be user "error" that it was even created in `doc/source`, rather than some more suitable location, simply because that's probably where the example code was run.  I presume we'd instead add a blanket default exclusion of all `sqlite` files.

I agree on closing this PR.  @Rishabhmishra328 feel free to address the concerns raised above, and this PR can be re-opened or a new one created.

**Comment by Rishabhmishra328** [comment]
Sorry for such a late reply,ill look into all the issues,please reopen the PR as this is my GSoC PR and would make my proposal invalid.

**Comment by dpshelio** [comment]
@Rishabhmishra328 that the PR is closed does not make your application invalid.  The PSF just ask for a sample code or contribution to the project. Nevertheless I will encourage you to keep contributing, whether submitting a new PR or updating this one is up to you. This can be opened when you commit these changes - though I would think that doing a new one will be easier (you can always reference to this one by `#1364` in the comments so whoever arrives to this one see the link to the new one).

**Comment by debugger22** [comment]
@Rishabhmishra328 You can add `#doctest: +SKIP` as a suffix to stop doctest from checking one specific line of command. This might come handy where outputs may differ depending on environment or time.
